app/views/render_async/_request.js.erb

```diff
@@ -1,7 +1,15 @@
 if (window.jQuery) {
   (function($) {
     $(document).ready(function() {
-      $.ajax({ url: "<%= path.html_safe %>" }).always(function(response) {
+      var headers = <%= headers.to_json.html_safe %>;
+      headers['X-CSRF-Token'] = document.querySelector('meta[name="csrf-token"]').content;
```

Comment:
I don't like CSRF token being sent on every request 🤔

Reply:
Do you think we can leave that out and encourage people to put CSRF token if they need one?

Reply:
I think that's good idea 👍

Reply:
What about only including it automatically if the request method is not a GET request.. since that's the only time it will be necessary anyway?

Reply:
@jhubert yes, I also like the things that "just work" out of the box. It makes sense to me to include it by default now!